src/PermitTokens/permitTokenAbi.json

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
Do we need both this and the Dai ABI? we could just keep 1

Reply:
dai implements permit differently so I think we need both